hknweb/candidate/templates/candidate/challenge_detail.html

```diff
@@ -19,23 +19,23 @@
     <b>Detailed Description:</b> {{ challenge.description }}<br/>
     <b>Proof:</b> {{ challenge.proof }}<br/>
     <b>Status:</b>
-    {% if not challenge.reviewed %}
-        <img src="{% static 'candidate/maybe.png'%}" alt="in progress" class="inline-icon">
-        Waiting for officer response
-    {% elif challenge.confirmed %}
+    {% if challenge.officer_confirmed is False or challenge.csec_confirmed is False %}
+        <img src="{% static 'candidate/no.png'%}" alt="declined" class="inline-icon">
+        Declined :<</br>
+    {% elif challenge.officer_confirmed is True and challenge.csec_confirmed is True %}
         <img src="{% static 'candidate/yes.png'%}" alt="confirmed" class="inline-icon">
-        Confirmed uwu
+        Confirmed uwu</br>
     {% else %}
-        <img src="{% static 'candidate/no.png'%}" alt="declined" class="inline-icon">
-        Declined :<</b>
-    {% endif %}<br/>
+        <img src="{% static 'candidate/maybe.png'%}" alt="in progress" class="inline-icon">
+        Waiting for officer/csec response</br>
+    {% endif %}
```

Comment:
Do you think it would be better to have two checkmarks, for officer / csec confirmation, or have one status (at the cost of code complexity)?

Reply:
The candidate doesn't need to know the details, but the officer needs to know